_data/landscape/meshes.yml

```diff
@@ -287,8 +287,8 @@
   tracing: "<i><a class='blue-grey-text' href='https://blog.turbinelabs.io/turbine-labs-is-shutting-down-and-our-team-is-joining-slack-2ad41554920c'>Project shutdown</a></i>"
   encryption: "<i><a class='blue-grey-text' href='https://blog.turbinelabs.io/turbine-labs-is-shutting-down-and-our-team-is-joining-slack-2ad41554920c'>Project shutdown</a></i>"
 
-- name: SOFAMesh
-  desc: A solution for large-scale Service Mesh based on Istio.
+- name: Open Service Mesh
+  desc: Open Service Mesh (OSM) is a lightweight and extensible cloud native service mesh.
```

Comment:
@VictoryWekwa Thanks! You can go ahead and delete lines to 92 through 301 as well. https://github.com/layer5io/layer5/pull/704/files#diff-c960221cf975c9a39633140c3e0f3d72R292-R301

Reply:
@leecalcote do you mean 92 or 192? 😮

Reply:
Okay. Will make the necessary corrections and create a pull request.
Should be 292 through 301.